machine_learning_tutorial.Rmd

```diff
@@ -0,0 +1,394 @@
+# Introduction of machine learning in R
+
+Feifan Li
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,warning = FALSE, message = FALSE)
+```
+
+```{r}
+library(caret)
+library(mlbench)
+library(naivebayes)
+library(rpart)
+library(randomForest)
+library(ggplot2)
+library(lattice)
+library(recipes)
+library(dplyr)
+library(AppliedPredictiveModeling)
+library(gridExtra)
+library(caTools)
+```
+## 1.Introduction
```

Comment:
Please remove the numbered headings such as "1.", here and below.  The ## and ### will all convert into numbered headings during the rendering.